projects/canopy/src/lib/filter-buttons/filter-group.component.ts

```diff
@@ -0,0 +1,153 @@
+import {
+  Component,
+  ContentChildren,
+  forwardRef,
+  HostBinding,
+  Input,
+  Optional,
+  QueryList,
+  Self,
+  ViewEncapsulation,
+  Host,
+  SkipSelf,
+  ContentChild,
+} from '@angular/core';
+import { ControlValueAccessor, NgControl, FormGroupDirective } from '@angular/forms';
+
+import { LgErrorStateMatcher } from '../forms/validation/error-state-matcher';
+import { LgValidationComponent } from '../forms/validation/validation.component';
+import { LgFilterButtonComponent } from './filter-button/filter-button.component';
+import { FilterVariant } from './filter-button.interface';
+import { LgDomService } from '../utils/dom.service';
+
+let nextUniqueId = 0;
+
+@Component({
+  selector: 'lg-filter-group',
+  templateUrl: './filter-group.component.html',
+  styleUrls: ['./filter-group.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class LgFilterGroupComponent implements ControlValueAccessor {
+  private _name = `lg-filter-group-${nextUniqueId++}`;
+
+  @Input() id = `lg-filter-group-id-${nextUniqueId++}`;
+  @Input() disabled = false;
+  @Input() ariaDescribedBy: string;
+  _variant: FilterVariant = 'select-one';
+  @Input()
+  set variant(variant: FilterVariant) {
```

Comment:
Just wondering why we're using a getter + setter on the variant input? It doesn't seem to be doing any logic.

Reply:
they were left behind after I tested something. Got rid of them. :)